### DISCUSSION TITLE
Nesting of objects in json::parse

### DISCUSSION THREAD

**Discussion by Kolhun** [nlohmann/json#4120] [Q&A]
Hello everyone, tell me there is a string, I translate it into JSON (json::parse). The question is that the keys I need are located in the "data" object how do I access the keys that are located in it? And work directly with it

![image](https://github.com/nlohmann/json/assets/36815965/69b0dad9-8eea-4886-b9bf-13fde32cc624)
![image](https://github.com/nlohmann/json/assets/36815965/5b5dab81-4e34-4460-aac8-6fe892d305be)

**Comment by Kolhun** [comment]
By the way, why does the "name" key return strange characters to me? When does the original line look like in the screenshot below? The idea should be Cyrillic
![image](https://github.com/nlohmann/json/assets/36815965/6f2165f2-65d5-4be3-adf8-db3e748abb15)

**Reply by nlohmann** [reply]
Make sure the input is UTF-8.

**Reply by Kolhun** [reply]
![image](https://github.com/nlohmann/json/assets/36815965/59730008-8fdc-473c-bccb-e40554d1f9ca)

**Reply by Kolhun** [reply]
Sorry, from the other plugin everything is ok, but from the json side will it be so
